pkg/core/interop/crypto/interop.go

```diff
@@ -0,0 +1,29 @@
+package crypto
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/core/interop"
+	"github.com/nspcc-dev/neo-go/pkg/vm"
+	"github.com/nspcc-dev/neo-go/pkg/vm/emit"
+)
+
+// GetInterop returns interop getter for crypto-related stuff.
+func GetInterop(ic *interop.Context) func(uint32) *vm.InteropFuncPrice {
+	return func(id uint32) *vm.InteropFuncPrice {
+		switch id {
+		case emit.InteropNameToID([]byte("Neo.Crypto.ECDsaVerify")):
```

Comment:
It involves hashing, we don't want to recalculate it again and again for every function invocation.

Reply:
Fixed.